member/src/test/scala/com/improving/app/member/api/MemberServiceIntegrationSpec.scala

```diff
@@ -1,46 +1,190 @@
 package com.improving.app.member.api
 
 import akka.actor.ActorSystem
-import com.improving.app.member.Main
-import kalix.scalasdk.testkit.KalixTestKit
+import akka.actor.typed.scaladsl.adapter.ClassicActorSystemOps
+import akka.event.{Logging, LoggingAdapter}
+import com.improving.app.member.domain.MemberStatus.{
+  MEMBER_STATUS_ACTIVE,
+  MEMBER_STATUS_INACTIVE,
+  MEMBER_STATUS_INITIAL,
+  MEMBER_STATUS_SUSPENDED,
+  MEMBER_STATUS_TERMINATED
+}
```

Comment:
Don't you just need all of them? Use ._ after MemberStatus ?

Reply:
Done